detox/package.json

```diff
@@ -123,6 +123,8 @@
       "__tests__",
       ".test.js$",
       ".mock.js$",
+      "index.js",
```

Comment:
The discussed logic, i.e. "use cached globals VS new require()", has zero meaning on unit tests level.

We cover it on integration and e2e tests level instead which test THE INTENT of this pull request, not the obvious implementation details.